.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Run unit tests
+on: [push]
+jobs:
+  run-unit-tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Install snpseq_metadata
+        run: "pip install -r requirements_dev.txt -e ."
+
+      - name: Download the ENA/SRA XML schema and generate python models
+        run: "bash ./snpseq_metadata/scripts/generate_python_models.sh xsdata"
+
+      - name: Run unit tests
+        run: "./validate-unit.sh"
+
+      - name: Run integration tests
+        run: |
+          mkdir python_out
+          tests/validate_test_data.sh "$(pwd)" "python_out"
```

Comment:
there was a test case for building and running the docker container as well but here you are removing that. is that intentional?

Reply:
Sorry, it went a little bit to fast :) I've added them now